osu.Game/Overlays/SettingsOverlay.cs

```diff
@@ -93,7 +96,11 @@ private void load(OsuGame game)
                         new SidebarButton
                         {
                             Section = section,
-                            Action = b => sectionsContainer.ScrollContainer.ScrollTo(b),
+                            Action = s =>
+                            {
+                                sectionsContainer.ScrollTo(s);
+                                sidebar.State = ExpandedState.Contracted;

```

Comment:
This feels super weird... You move onto an icon to click it, suppose you wait around 750ms so it starts expanding, click, it contracts, then 750ms later it expands again.

Kinda preferred the feel before?